zebra-consensus/src/block.rs

```diff
@@ -271,13 +283,29 @@ where
                 })?;
             }
 
+            // TODO: Add link to lockbox stream ZIP
+            let expected_deferred_amount = subsidy::funding_streams::funding_stream_values(
```

Comment:
It would be nice to get this amount from `subsidy_is_valid()` in a follow-up PR for cleanups.

Reply:
I think an even better place might be `check::miner_fees_are_valid`. I'll try doing it tomorrow.